grpcurl_test.go

```diff
@@ -50,7 +50,7 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		panic(err)
 	}
-	sourceProtoFiles, err = DescriptorSourceFromProtoFiles(nil, "../../../google.golang.org/grpc/interop/grpc_testing/test.proto")
+	sourceProtoFiles, err = DescriptorSourceFromProtoFiles(nil, "testing/test.proto")
```

Comment:
When building with Go 1.11 with `GO111MODULE=on` environment set, we can no longer rely on the file layout of dependencies all under a single `GOPATH` directory, so this relative reference to the gRPC project doesn't work. To work around (since it's just for a test anyway), I've copied/forked this `test.proto` file into this repo.